clowdapp.yaml

```diff
@@ -74,6 +74,16 @@ objects:
                 optional: true
           - name: BYPASS_UNLEASH
             value: ${BYPASS_UNLEASH}
+          - name: GITHUB_OWNER
+            value: "RedHatInsights"
+          - name: GITHUB_REPO
+            value: "ros-backend"
+          - name: GITHUB_DEPLOYMENT_BRANCH
+            value: "main"
+          - name: GITHUB_API_PAGE_SIZE
+            value: "5"
+          - name: GITHUB_API_MAXIMUM_RETRIES
+            value: "3"
```

Comment:
You'll need to define these as parameters at the end of this file, like so

```
- description: organisation/owner-name of the GitHub repo
  name: GITHUB_OWNER
  value: "RedHatInsights"
```

Then use those here,

```
- name: GITHUB_OWNER
  value: ${GITHUB_OWNER}
```
I suggest adding these after `Feature flags` vars in a new section at the end.